### PR TITLE
We only new a handfull of threads for the httpserver in the clusterco…

### PR DESCRIPTION
**Pull request by baldersheim**
…ntroller and metricsproxy.

@bjorncs PR

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/container/http/JettyHttpServer.java [hunk 1]
```diff
@@ -82,10 +75,17 @@ public void getConfig(ServerConfig.Builder builder) {
 
     private void configureJettyThreadpool(ServerConfig.Builder builder) {
         if (cluster == null) return;
+        if (cluster instanceof ApplicationContainerCluster) {
+            configureApplicationClusterJettyThreadPool(builder);
+        } else {
+            builder.minWorkerThreads(2).maxWorkerThreads(4);
```
Comment (bjorncs) [inline config-model/src/main/java/com/yahoo/vespa/model/container/http/JettyHttpServer.java:81]: Have you tried this? I'm pretty sure 2-4 is too low.

**Review by bjorncs — APPROVED** [review]

File: jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.yahoo.container.logging;// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 import com.yahoo.test.json.JsonTestHelper;
-import org.junit.jupiter.api.Test;
+import org.junit.Test;
```
Comment (bjorncs) [inline jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java:4]: You are effectively disabling all tests here now

**Review by bjorncs — CHANGES_REQUESTED** [review]

File: jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java (hunk 2 above)
Comment (bjorncs) [reply jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java:4]: FYI `org.junit.jupiter.api` is junit5 while `org.junit` is junit4.

File: config-model/src/main/java/com/yahoo/vespa/model/container/http/JettyHttpServer.java (hunk 1 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/vespa/model/container/http/JettyHttpServer.java:81]: No, not tried. Will run a few system tests.

File: jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java (hunk 2 above)
Comment (baldersheim) [reply jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java:4]: This was not intended. That was just a temporary thing to get intellij to run tests in config-model. It did not recognize the jupiter.api.

**Review by bjorncs — APPROVED** [review]

**Pull request merged**